src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1142,6 +1142,14 @@
     <value>Use the new text renderer ("AtlasEngine")</value>
     <comment>{Locked="AtlasEngine"}</comment>
   </data>
+  <data name="Profile_ReloadEnvVars.Header" xml:space="preserve">
+    <value>Reload environment variables</value>

```

Comment:
This feels like it has steeped too long with the rest of our language. I can't figure out why I feel like that.

I want to suggest something that doesn't need as long of a sub-heading and is more punchy like "Launch this application with a new environment block" or "Launch this application with a clean environment" or something.

Alternatively, we could rephrase it as a negative and flip the sense.

"[COMPATIBILITY] Inherit environment variables"

(This one would need a help-text that explains why it doen't guarantee clean state)